snc-library.sh

```diff
@@ -202,7 +202,7 @@ function renew_certificates() {
     start_vm ${vm_prefix}
 
     # After cluster starts kube-apiserver-client-kubelet signer need to be approved
-    timeout 300 bash -c -- "until ${OC} get csr | grep Pending; do echo 'Waiting for first CSR request.'; sleep 2; done"
+    timeout 500 bash -c -- "until ${OC} get csr | grep Pending; do echo 'Waiting for first CSR request.'; sleep 2; done"
```

Comment:
Can you change this to `500s` to avoid the need to check the manpage? Otherwise, nothing against it if this changed is required for you for a successful run. 5 minutes timeout or 8min20s timeout are not going to make a significant difference

Reply:
sure, will add time units to make it clear :+1:

Reply:
done